tests/Integration/Behaviour/Features/Context/OrderFeatureContext.php

```diff
@@ -188,11 +190,66 @@ public function checkOrderDiscount($position, $discountTaxIncluded, $discountTax
         }
     }
 
+    /**
+     * @Then created order should have :quantity products in total
```

Comment:
`created order` is a bit vague, can we give it a reference and use it everywhere ? Reference can be artificial, it's just an explicit way to target the order. See https://github.com/PrestaShop/PrestaShop/blob/develop/tests/Integration/Behaviour/Features/Context/CustomerManagerFeatureContext.php#L112